src/index.html

```diff
@@ -6,13 +6,62 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      />
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__subtitle">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="card__reviews-stars">
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+        </div>
+        <div class="card__reviews-text">Reviews: 5</div>
+      </div>
+
+      <div class="card__prices">
```

Comment:
The section that contains the price can be more semantically represented by using a `dl` list with `dt` and `dd` tags for the term 'Price' and the value respectively.